documentation/asciidoc/topics/proc_restart.adoc

```diff
@@ -1,8 +1,13 @@
 [id='restarting-{context}']
 = Restarting {brandname} Clusters
+Complete the following procedure to restart {brandname} clusters after shutdown.
 
-* Set the value of `spec.replicas` to the same number of nodes that were in the
-cluster before you shut it down.
+.Procedure
+
+* Set the value of `spec.replicas` to an appropriate number of nodes for your cluster.
+
+For {datagridservice} nodes, the value of `spec.replicas` must be the same
```

Comment:
Looks like we should also notify customers, that they are not needed to remember number of replicas and can get it from status.replicasWantedAtRestart

Reply:
@dmvolod is `status.replicasWantedAtRestart` available from `kubectl get infinispan -o yaml`?

Reply:
Yes, @oraNod sure
kubectl get infinispan cr-name -o=jsonpath='{.status.replicasWantedAtRestart}'

Reply:
@rigazilla if we need to have replicas = replicasWantedAtRestart in some cases, can we set this option automatically without user notation as implemented now?

Reply:
@dmvolod yeah sound very reasonable

Reply:
@dmvolod and @rigazilla do we need to create a separate issue to set `replicasWantedAtRestart` automatically?

Reply:
@oraNod, I think a separate issue is better. so we don't mix too many things here

Reply:
https://github.com/infinispan/infinispan-operator/issues/434